### PR TITLE
[nrf528xx] update platforms README.md

### PR DESCRIPTION
**Pull request by LuDuda**

### CODE REVIEW

File: examples/platforms/nrf52811/README.md [hunk 1]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
 
-The RTT output can be viewed in the J-Link RTT Viewer, which is available from SEGGER.
+You can set the desired log level by using the `OPENTHREAD_CONFIG_LOG_LEVEL` define.
 
-The viewer is also included in the nRF Tools. 
-To read or write messages over RTT:
+To enable the highest verbosity level, use the following command when building:
+```
+$ make -f examples/Makefile-nrf52811 FULL_LOGS=1
+```
```
Comment (KaSroka) [inline examples/platforms/nrf52811/README.md:224]: User should append `FULL_LOGS=1` to other flags. This suggests to build OT with this flag only.

File: examples/platforms/nrf52811/README.md [hunk 2]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+JLinkRTTClient -RTTTelnetPort 19021
+```
+
+### Mass Storage Device
+
+Due to a known issue in Segger’s J-Link firmware, depending on your version, you might experience data corruption or drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device.
+For that you can use both Windows and Linux:
+
+### Disabling the Mass Storage Device on Windows
```
Comment (KaSroka) [inline examples/platforms/nrf52811/README.md]:
```suggestion
#### Disabling the Mass Storage Device on Windows
```

File: examples/platforms/nrf52811/README.md [hunk 3]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+2. Run `J-Link Commander`. The configuration window appears.
+3. From the Specify Target Device dropdown menu, select `NRF52810_XXAA`.
+4. From the Target Interface & Speed dropdown menu, select `SWD`.
+5. Run the following command:
+```
+MSDDisable
+```
+6. Power cycle the DK.
+
+### Disabling the Mass Storage Device on Linux
```
Comment (KaSroka) [inline examples/platforms/nrf52811/README.md]:
```suggestion
#### Disabling the Mass Storage Device on Linux
```

File: examples/platforms/nrf52811/README.md [hunk 4]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+
+### Hardware Flow Control detection
+
+By default, J-Link detects at runtime if the target uses flow control or not.
+
+Automatic HWFC detection is done by driving P0.07 (Clear to Send (CTS)) from the interface MCU and evaluating the state of P0.05 (Request to Send (RTS)) when the first data is sent or received. If the state of P0.05 (RTS) is high, HWFC is assumed not to be used.
+
+To avoid potential race conditions, it is possible to force hardware flow control and bypass runtime auto-detection. For that you can use both Windows and Linux:
+
+### Disabling the HWFC detection on Windows
```
Comment (KaSroka) [inline examples/platforms/nrf52811/README.md]:
```suggestion
#### Disabling the HWFC detection on Windows
```

File: examples/platforms/nrf52811/README.md [hunk 5]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+2. Run `J-Link Commander`. The configuration window appears.
+3. From the Specify Target Device dropdown menu, select `NRF52810_XXAA`.
+4. From the Target Interface & Speed dropdown menu, select `SWD`.
+5. Run the following command:
+```
+SetHWFC Force
+```
+6. Power cycle the DK.
+
+### Disabling the HWFC detection on Linux
```
Comment (KaSroka) [inline examples/platforms/nrf52811/README.md]:
```suggestion
#### Disabling the HWFC detection on Linux
```

**Review by KaSroka — APPROVED** [review]
Looks good, just few nits!

File: examples/platforms/nrf52811/README.md [hunk 6]
```diff
@@ -2,28 +2,36 @@
 
 This SoC is meant to be used in the configuration that involves the Host Processor and the IEEE 802.15.4 radio.
 In this configuration, the full OpenThread stack is running on the Host Processor and the nRF52811 SoC acts as an IEEE 802.15.4 radio.
-The radio is running a minimal OpenThread implementation that allows for communication between the Host Processor and the nRF52811. In this architecture the nRF52811 SoC device is called NCP radio or RCP (Radio Co-Processor).
+The radio is running a minimal OpenThread implementation that allows for communication between the Host Processor and the nRF52811.
+In this architecture the nRF52811 SoC device is called NCP radio or RCP (Radio Co-Processor).
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

In this architecture, the nRF52811 SoC device is called NCP radio or RCP (Radio Co-Processor).

File: examples/platforms/nrf52811/README.md [hunk 7]
```diff
@@ -2,28 +2,36 @@
...
 
 The nRF52811 platform is currently under development.
 
 For the SoC capable to a run full OpenThread stack please check the [nRF52840 platform][nRF52840-page].
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

For the SoC capable to a run full OpenThread stack, see the [nRF52840 platform][nRF52840-page].

File: examples/platforms/nrf52811/README.md [hunk 8]
```diff
@@ -2,28 +2,36 @@
...
 
 [nRF52840-page]: ./../nrf52840/README.md
 
-## Toolchain
+## Prerequisites
+
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md:19]: Add a sentence that describes this section. This is required to let the reader know what to expect in this section.

For example:

Before you start building the examples, you must download and install the toolchain and the tools required for flashing and debugging.

File: examples/platforms/nrf52811/README.md [hunk 9]
```diff
@@ -2,28 +2,36 @@
...
 ```bash
 $ cd <path-to-openthread>
 $ ./script/bootstrap
 ```
 
+### Flashing and debugging tools
+
+[nRF5-Command-Line-Tools]: https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF5-Command-Line-Tools
+
+In order to flash, debug and make use of logging features on nRF52811 DK, please install the [nRF5 Command Line Tools][nRF5-Command-Line-Tools].
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

Install the [nRF5 Command Line Tools][nRF5-Command-Line-Tools] to flash, debug, and make use of logging features on the nRF52811 DK.

File: examples/platforms/nrf52811/README.md [hunk 10]
```diff
@@ -2,28 +2,36 @@
...
+
 ## Building the examples
 
 With this platform you can build:
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

With this platform, you can build:

Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit line 55, add the missing `:

You can convert them to hex using `arm-none-eabi-objcopy`:

File: examples/platforms/nrf52811/README.md [hunk 11]
```diff
@@ -2,28 +2,36 @@
...
 $ ./script/bootstrap
 ```
 
+### Flashing and debugging tools
+
+[nRF5-Command-Line-Tools]: https://www.nordicsemi.com/Software-and-Tools/Development-Tools/nRF5-Command-Line-Tools
+
+In order to flash, debug and make use of logging features on nRF52811 DK, please install the [nRF5 Command Line Tools][nRF5-Command-Line-Tools].
+
 ## Building the examples
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md:41]: Below we have "Building the examples with native SPI Slave support". This makes this "Building the examples" section confusing. Use one of the following approaches:
- specify how we're building the examples here ("without native SPI Slave support" or "with default procedure"), or
- make "Building the examples with native SPI Slave support" a subsection of "Building the examples"

I prefer the first approach, as it is easier to implement, but you're the SME here, so it's up to you.

File: examples/platforms/nrf52811/README.md [hunk 12]
```diff
@@ -104,9 +112,7 @@ Copy the files to /usr/bin for easier access.
 ## Flashing binaries
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md:99]: Edit for line 98:

### Building host executables

(remove "the" for consistency with line 61)

File: examples/platforms/nrf52811/README.md (hunk 9 above)
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Maybe it would make sense to mention SEGGER J-Link here? 

For example:

Install the [nRF5 Command Line Tools][nRF5-Command-Line-Tools] to flash, debug, and make use of logging features on the nRF52811 DK with SEGGER J-Link.

File: examples/platforms/nrf52811/README.md [hunk 13]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
 
 [CLI]: ./../../../src/cli/README.md
 
-## Working with the logging module
+## Segger J-Link tools
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: As far as I see, we're only discussing SEGGER J-Link in this section. So this level 2 section is not needed. Moreover, it doesn't have any description before the subsection.

So `### RTT logging` should become `## Working with SEGGER J-Link for RTT logging` or something like this. Remember to update the header levels in the subsections if you go for this edit.

File: examples/platforms/nrf52811/README.md [hunk 14]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+```
 
-1. Connect an nRF5 development board via with a USB cable. 
-2. Run the J-Link RTT Viewer.
-3. In the Specify Target Device dropdown menu, select the nRF52 device.
-4. In the Target Interface & Speed dropdown menu, select the SWD interface.
+You can use the logging module on both Windows and Linux.
 
-The intended log level can be set using `OPENTHREAD_CONFIG_LOG_LEVEL` define.
+#### Logging on Windows
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Add a sentence to introduce the steps:

`To enable the RTT logging module on Windows:`

File: examples/platforms/nrf52811/README.md [hunk 15]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
 
-## Disabling the Mass Storage Device
+1. Connect the DK to your machine with a USB cable.
+2. Run `J-Link RTT Viewer`. The configuration window appears.
+3. From the Specify Target Device dropdown menu, select `NRF52810_XXAA`.
+4. From the Target Interface & Speed dropdown menu, select `SWD`.
 
-Depending on your software version, a known issue in SEGGER's J-Link firmware can cause data corruption or data drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device:
+#### Logging on Linux
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Add a sentence to introduce the steps:

`To enable the RTT logging module on Linux:`

File: examples/platforms/nrf52811/README.md [hunk 16]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+JLinkExe -device NRF52810_XXAA -if SWD -speed 4000 -autoconnect 1 -SelectEmuBySN <SEGGER_ID> -RTTTelnetPort 19021
+```
+3. Run `JLinkRTTTelnet` to obtain the RTT logs from the connected device in a separate console. For example:
+```
+JLinkRTTClient -RTTTelnetPort 19021
+```
+
+### Mass Storage Device
+
+Due to a known issue in Segger’s J-Link firmware, depending on your version, you might experience data corruption or drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device.
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

Depending on your version, due to a known issue in SEGGER's J-Link firmware, you might experience data corruption or data drops if you use the serial port. You can avoid this issue by disabling the Mass Storage Device.

File: examples/platforms/nrf52811/README.md [hunk 17]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+For that you can use both Windows and Linux:
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: This sentence is not needed. Please remove.

File: examples/platforms/nrf52811/README.md (hunk 2 above)
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Add a sentence to introduce the steps:

To disable the Mass Storage Device on Windows:

(also applies to Linux in line 284)

File: examples/platforms/nrf52811/README.md [hunk 18]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+2. Run `JLinkExe` to connect to the target. For example:
+```
+JLinkExe -device NRF52810_XXAA -if SWD -speed 4000 -autoconnect 1 -SelectEmuBySN <SEGGER_ID> -RTTTelnetPort 19021
+```
+3. Run `JLinkRTTTelnet` to obtain the RTT logs from the connected device in a separate console. For example:
+```
+JLinkRTTClient -RTTTelnetPort 19021
+```
+
+### Mass Storage Device
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Is this section added due to the known issue only, and if there wasn't any known issue you wouldn't touch the MSD?
If so, I would rephrase: `### Mass Storage Device known issue`

File: examples/platforms/nrf52811/README.md [hunk 19]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+```
+3. Run the following command:
+```
+MSDDisable
+```
+4. Power cycle the DK.
+
+### Hardware Flow Control detection
+
+By default, J-Link detects at runtime if the target uses flow control or not.
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

By default, SEGGER J-Link automatically detects at runtime whether the target is using Hardware Flow Control (HWFC).

File: examples/platforms/nrf52811/README.md [hunk 20]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+
+Automatic HWFC detection is done by driving P0.07 (Clear to Send (CTS)) from the interface MCU and evaluating the state of P0.05 (Request to Send (RTS)) when the first data is sent or received. If the state of P0.05 (RTS) is high, HWFC is assumed not to be used.
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit -- note that I'm unsure about the meaning of "is assumed not to be used", and this edit assumes that HWFC is not used at the moment when the state of P0.05 is evaluated; if the meaning should refer to a future usage, use `it is assumed that HWFC is not to be used`:

The automatic HWFC detection is done by driving P0.07 (Clear to Send - CTS) from the interface MCU and evaluating the state of P0.05 (Request to Send - RTS) when the first data is sent or received. If the state of P0.05 (RTS) is high, it is assumed that HWFC is not used.

File: examples/platforms/nrf52811/README.md [hunk 21]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+
+To avoid potential race conditions, it is possible to force hardware flow control and bypass runtime auto-detection. For that you can use both Windows and Linux:
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Edit:

To avoid potential race conditions, you can force HWFC and bypass the runtime auto-detection.

File: examples/platforms/nrf52811/README.md (hunk 4 above)
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Add a sentence that introduces the steps:

To force HWFC and bypass the runtime auto-detection on Windows:

(same goes for Linux in line 315)

File: examples/platforms/nrf52811/README.md [hunk 22]
```diff
@@ -222,29 +228,103 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
+```
+3. Run the following command:
+```
+SetHWFC Force
+```
+4. Power cycle the DK.
 
- - On Linux or macOS (OS X), open JLinkExe from the terminal and run the command `MSDDisable`.
- - On Microsoft Windows, open the J-Link Commander application and run the command `MSDDisable`.
+You can find more details [here][J-Link_OB].
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md]: Adding links under `here` is not great: the reader doesn't know what to expect on the other side of the link.

Edit:

For details, see [<name of the page>][J-Link_OB].

I cannot find the link. Also, don't use the underscore in the <name of the page>.

File: examples/platforms/nrf52840/README.md [hunk 23]
```diff
@@ -1,27 +1,38 @@
 # OpenThread on nRF52840 Example
 
-This directory contains example platform drivers for [Nordic Semiconductor nRF52840 SoC][nRF52840].
+<a href="http://threadgroup.org/technology/ourtechnology#certifiedproducts">
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:3]: Do not start a page with an image. Rather, move it to after line 7.

File: examples/platforms/nrf52840/README.md [hunk 24]
```diff
@@ -1,27 +1,38 @@
...
+This directory contains example platform drivers for [Nordic Semiconductor nRF52840 SoC][nRF52840]. The OpenThread stack has been officially certified as a *Thread Certified Component* on this platform.
+
+[nRF52840]: https://www.nordicsemi.com/Products/Low-power-short-range-wireless/nRF52840
 
 To facilitate Thread products development with the nRF52840 platform, Nordic Semiconductor provides <i>nRF5 SDK for Thread and Zigbee</i>. See [Nordic Semiconductor's nRF5 SDK for Thread and Zigbee][nRF5-SDK-section] section for more details.
 
 [nRF5-SDK-section]: #nordic-semiconductors-nrf5-sdk-for-thread-and-zigbee
 
-## Toolchain
+## Prerequisites
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:15]: Apply the relevant edits from nRF52811 file in this file as well.

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: examples/platforms/nrf52811/README.md (hunk 13 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md]: I would leave this section since it clearly gathers all Segger related stuff. I will try to add some description.

File: examples/platforms/nrf52811/README.md (hunk 1 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md:224]: that's true. The other sentences actually are in the same fasion. I will change just that. Let's assume that `make` is known for our customer.

File: examples/platforms/nrf52811/README.md (hunk 14 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md]: The previous subsection introduces those two chapters.

File: examples/platforms/nrf52811/README.md (hunk 15 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md]: Same comment as above.

File: examples/platforms/nrf52811/README.md (hunk 4 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md]: I don't feel this is needed. The section greatly described the purpose. The introduction has been made in section above? Thoughts?

File: examples/platforms/nrf52840/README.md (hunk 23 above)
Comment (LuDuda) [reply examples/platforms/nrf52840/README.md:3]: Hmm, it looks much better here :) On the preview you can see that thi image is right aligned.

**Comment by LuDuda** [comment]
Thanks for comments. I think i applied all your suggestions which i agree with.

File: examples/platforms/nrf52840/README.md (hunk 23 above)
Comment (greg-fer) [reply examples/platforms/nrf52840/README.md:3]: Ok, it looks fine.

File: examples/platforms/nrf52811/README.md [hunk 25]
```diff
@@ -222,29 +216,106 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
```
Comment (greg-fer) [inline examples/platforms/nrf52811/README.md:207]: Break down the step 8 in Running the example into 3 steps:

8. Open a terminal connection on the second board and attach a node to the network.
   The CLI MTD example uses the direct UART connection.
9. Start a terminal emulator like screen and connect to the used COM port with the following UART settings:
    - Baud rate: 115200
    - 8 data bits
    - 1 stop bit
    - No parity
    - HW flow control: RTS/CTS
This allows you to view the raw UART output.
10. Run the following command to connect to the second board.
 ```bash
 screen /dev/ttyACM1 115200
 ```
Now you are connected with the CLI.

File: examples/platforms/nrf52840/README.md [hunk 26]
```diff
@@ -53,17 +65,17 @@ The driver can be found in `third_party/NordicSemiconductor/libraries/usb/nordic
...
 See [nRF52840 Dongle Programming][nrf52840-dongle-programming] for more details about how to use the USB bootloader.
 
-[nrf52840-dongle-programming]: https://infocenter.nordicsemi.com/index.jsp?topic=%2Fcom.nordic.infocenter.nrf52%2Fdita%2Fnrf52%2Fdevelopment%2Fnrf52840_dongle%2Fprogramming.html&cp=2_0_4_4
+[nrf52840-dongle-programming]: https://www.nordicsemi.com/DocLib/Content/User_Guides/nrf52840_dongle/latest/UG/nrf52840_Dongle/programming
 
-### Native SPI Slave support
+### Native SPI support
 
 You can build the libraries with support for native SPI Slave.
 To build the libraries, run make with the following parameter:
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:81]: Edit for lines 87-89:

With this option enabled, the NCP example can communicate through SPI with wpantund (provided that the wpantund host supports SPI Master). To achieve this communication, choose the right SPI device in the wpantund configuration file, `/etc/wpantund.conf`. See the following example.

Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:81]: Edits for lines 94-96:

In this example, [spi-hdlc-adapter][spi-hdlc-adapter] is a tool that you can use to communicate between NCP and wpantund over SPI. For this example, `spi-hdlc-adapter` is installed in `/usr/bin`.

File: examples/platforms/nrf52840/README.md [hunk 27]
```diff
@@ -53,17 +65,17 @@ The driver can be found in `third_party/NordicSemiconductor/libraries/usb/nordic
 
 You can build the libraries with support for USB bootloader with automatic USB DFU trigger support in PCA10059. As this dongle uses the native USB support, you must enable it as well.
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md]: You can build the libraries with support for the USB bootloader with USB DFU trigger support in PCA10059.

File: examples/platforms/nrf52840/README.md [hunk 28]
```diff
@@ -110,9 +122,7 @@ $ make -f examples/Makefile-nrf52840 DISABLE_CC310=1
 ## Flashing the binaries
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:120]: Edit for line 115:

Optional disabling of CryptoCell 310 support

The previous header is "optional", and I assume this one is too, so let's make it clear this is not mandatory.

File: examples/platforms/nrf52840/README.md [hunk 29]
```diff
@@ -110,9 +122,7 @@ $ make -f examples/Makefile-nrf52840 DISABLE_CC310=1
...
 
 Flash the compiled binaries onto nRF52840 using `nrfjprog` which is
-part of the [nRF5x Command Line Tools][nRF5x-Command-Line-Tools].
-
-[nRF5x-Command-Line-Tools]: https://www.nordicsemi.com/eng/Products/nRF52840#Downloads
+part of the [nRF5 Command Line Tools][nRF5-Command-Line-Tools].
 
 ```bash
 $ nrfjprog -f nrf52 --chiperase --program output/nrf52840/bin/ot-cli-ftd.hex --reset
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:126]: Please apply changes from the 811 file to the Running the example section, including but not limited to:

To test the example:
1. Prepare two boards with the flashed `CLI Example` (as shown above).
2. As the CLI example uses UART connection, start a terminal
   emulator like PuTTY and connect to the used COM port with the following UART settings:
    - Baud rate: 115200
    - 8 data bits
    - 1 stop bit
    - No parity
    - HW flow control: RTS/CTS
   On Linux system a port name should be called e.g. `/dev/ttyACM0` or `/dev/ttyACM1`.
This allows you to view the raw UART output.
3. Open a terminal connection on the first board and start a new Thread network.
 ```bash
 > panid 0xabcd
 Done
 > ifconfig up
 Done
 > thread start
 Done
 ```
After a couple of seconds the node becomes a Leader of the network.
 ```bash
 > state
 Leader
 ```
4. Open a terminal connection on the second board and attach a node to the network.
 ```bash
 > panid 0xabcd
 Done
 > ifconfig up
 Done
 > thread start
 Done
 ```
After a couple of seconds the second node attaches and becomes a Child.
 ```bash
 > state
 Child
 ```
5. List all IPv6 addresses of the first board.
 ```bash
 > ipaddr
 fdde:ad00:beef:0:0:ff:fe00:fc00
 fdde:ad00:beef:0:0:ff:fe00:9c00
 fdde:ad00:beef:0:4bcb:73a5:7c28:318e
 fe80:0:0:0:5c91:c61:b67c:271c
 ```
8. Choose one of them and send an ICMPv6 ping from the second board.
 ```bash
 > ping fdde:ad00:beef:0:0:ff:fe00:fc00
 16 bytes from fdde:ad00:beef:0:0:ff:fe00:fc00: icmp_seq=1 hlim=64 time=8ms
 ```
For a list of all available commands, visit [OpenThread CLI Reference README.md][CLI].

File: examples/platforms/nrf52840/README.md [hunk 30]
```diff
@@ -187,62 +197,106 @@ For a list of all available commands, visit [OpenThread CLI Reference README.md]
...
 ```
+4. Power cycle the DK.
 
-You can find more details [here][J-Link_OB].
+You can find more details [here][J-Link-OB].
 
-[J-Link_OB]: https://wiki.segger.com/J-Link_OB_SAM3U_NordicSemi#Hardware_flow_control_support
+[J-Link-OB]: https://wiki.SEGGER.com/J-Link_OB_SAM3U_NordicSemi#Hardware_flow_control_support
 
 ## Diagnostic module
```
Comment (greg-fer) [inline examples/platforms/nrf52840/README.md:317]: Edit for line 324:

- `704511c` - 18.09.2018 (the latest checked)

**Review by greg-fer — CHANGES_REQUESTED** [review]

File: examples/platforms/nrf52811/README.md (hunk 25 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md:207]: Thanks. I will combine step 9 and 10 since they are about the same.

File: examples/platforms/nrf52840/README.md (hunk 29 above)
Comment (LuDuda) [reply examples/platforms/nrf52840/README.md:126]: I don't fully understand this comment. I did some changes to align both instructions, however they are fundamentally different so i can't just copy/paste.

File: examples/platforms/nrf52811/README.md (hunk 25 above)
Comment (greg-fer) [reply examples/platforms/nrf52811/README.md:207]: They are clearly two different steps, so this would basically contradict the reason behind my edit. Whenever we use steps, we go for "1 step = 1 action", which also means "1 action = 1 imperative verb".

File: examples/platforms/nrf52840/README.md (hunk 29 above)
Comment (greg-fer) [reply examples/platforms/nrf52840/README.md:126]: I understand. My edit is more about how the steps are broken, not the contents of the procedure.

File: examples/platforms/nrf52811/README.md (hunk 25 above)
Comment (LuDuda) [reply examples/platforms/nrf52811/README.md:207]: Yes but step 10 is just execution of description in stage 9. So they should be combined, right?

Comment (greg-fer) [reply examples/platforms/nrf52811/README.md:207]: OK, it's one huge step then. What about:

8. Open a terminal connection on the second board and attach a node to the network:
   a. Start a terminal emulator like screen.  
   b. Connect to the used COM port with the following direct UART settings:
   * Baud rate: 115200
   * 8 data bits
   * 1 stop bit
   * No parity
   * HW flow control: RTS/CTS
     This allows you to view the raw UART output.
  c. Run the following command to connect to the second board.
```shell
screen /dev/ttyACM1 115200
```
You are now connected with the CLI.

**Comment by LuDuda** [comment]
I believe i applied fixes for all the comments.

https://github.com/LuDuda/openthread/blob/946774725116bfee41bb0350857217c4128bf24f/examples/platforms/nrf52811/README.md
https://github.com/LuDuda/openthread/blob/946774725116bfee41bb0350857217c4128bf24f/examples/platforms/nrf52840/README.md

**Review by greg-fer — APPROVED** [review]

**Comment by LuDuda** [comment]
I believe i applied fixes for all the comments.

https://github.com/LuDuda/openthread/blob/382256bdaef0b714350fe00431b32565c809c528/examples/platforms/nrf52811/README.md
https://github.com/LuDuda/openthread/blob/382256bdaef0b714350fe00431b32565c809c528/examples/platforms/nrf52840/README.md
